zubhub_frontend/zubhub/src/assets/css/index.css

```diff
@@ -596,6 +596,16 @@ html,
   display: none !important;
 }
 
-.MuiInputBase-root #phone{
+.MuiInputBase-root #phone {
   width: 80%;
-}
\ No newline at end of file
+}
+.MuiGrid-root #sixCenter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.MuiGrid-root #eightCenter{
```

Comment:
Remove these styles since these are not used anymore

Reply:
okay